### ISSUE TITLE
AWS Deploy Script

### ISSUE THREAD

**Issue by cycomachead** [beautyjoy/beautyjoy.github.io#19]
Depends on #17 

- create a `production` branch
- when a branch is merged into production, push the site to S3

**Comment by cycomachead** [comment]
https://circleci.com/docs/continuous-deployment-with-amazon-s3/

**Comment by cycomachead** [comment]
Aws.CS10.org is working off master, it's easy to change the config in CI
